specification/translation/Azure.AI.DocumentTranslation/models.tsp

```diff
@@ -484,3 +486,160 @@ model StartTranslationBody {
+    """)
+  @maxValue(100)
+  @minValue(1)
+  @query("maxpagesize")
+  maxpagesize?: int32 = 50;
+
+  @doc("Ids to use in filtering")
+  @clientName("translationIds")
+  @query("ids")
+  ids?: uuid[];
+
+  @doc("Statuses to use in filtering")
+  @query("statuses")
+  statuses?: string[];
+
+  @doc("the start datetime to get items after")
+  @query("createdDateTimeUtcStart")
+  createdDateTimeUtcStart?: utcDateTime;
+
+  @doc("the end datetime to get items before")
+  @query("createdDateTimeUtcEnd")
+  createdDateTimeUtcEnd?: utcDateTime;
+
+  @doc("the sorting query for the collection (ex: 'CreatedDateTimeUtc asc','CreatedDateTimeUtc desc')")
+  @query("orderby")
+  orderby?: string[];
+}
+
+@doc("Get Documents Status options")
+model GetDocumentsStatusOptions {
+  @doc("Format - uuid.  The operation id")
+  @clientName("translationId")
+  @path
+  id: uuid;
+
+  @doc("""
+    top indicates the total number of records the user wants to be returned across
+    all pages.
+                
+    Clients MAY use top and skip query parameters to
+    specify a number of results to return and an offset into the collection.
+    When
+    both top and skip are given by a client, the server SHOULD first apply skip
+    and then top on the collection.
+                
+    Note: If the server can't honor
+    top and/or skip, the server MUST return an error to the client informing
+    about it instead of just ignoring the query options.
+    """)
+  @maxValue(2147483647)
+  @query("top")
+  top?: int32;
+
+  @doc("""
+    skip indicates the number of records to skip from the list of records held by
+    the server based on the sorting method specified.  By default, we sort by
+    descending start time.
+                
+    Clients MAY use top and skip query
+    parameters to specify a number of results to return and an offset into the
+    collection.
+    When both top and skip are given by a client, the server SHOULD
+    first apply skip and then top on the collection.
+                
+    Note: If the
+    server can't honor top and/or skip, the server MUST return an error to the
+    client informing about it instead of just ignoring the query options.
+    """)
+  @maxValue(2147483647)
+  @query("skip")
+  skip?: int32;
+
+  @doc("""
+    maxpagesize is the maximum items returned in a page.  If more items are
+    requested via top (or top is not specified and there are more items to be
+    returned), @nextLink will contain the link to the next page.
+                
+    
+    Clients MAY request server-driven paging with a specific page size by
+    specifying a maxpagesize preference. The server SHOULD honor this preference
+    if the specified page size is smaller than the server's default page size.
+    """)
+  @maxValue(100)
+  @minValue(1)
+  @query("maxpagesize")
+  maxpagesize?: int32 = 50;
+
+  @doc("Ids to use in filtering")
+  @clientName("documentIds")
```

Comment:
Now that we are using an actual model, please move the renames to client.tsp and remove the client generator core import from this file.

Reply:
Updated